README.md

```diff
@@ -1,9 +1,13 @@
 ## Docker container solution for Spryker deploy
 There is Dockerfile for build Yves and Zed docker containers and additional services container configurations for run Spyrker shop suite.
 ## Quick installation
+## Create a new SSH key pair:
+Linux/MacOS:
 ```
 ssh-keygen -b 2048 -t rsa -f jenkins/id_rsa -q -N ""
 ```
+Instructions for windows users: https://goo.gl/xHZVUL
```

Comment:
Could you please add the documentation inside the README file, not 3rd party URL?